GameEngine/Players/EpsilonGreedyPlayer.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace GameEngine.Players
+{
+    public class EpsilonGreedyPlayer: GreedyPlayer
+    {
+        private static readonly Random Random = new Random();
+        private double Epsilon { get; set; }
```

Comment:
💡 Is this setter necessary? I can't find it used anywhere.

Reply:
Removed!